app/controllers/application_controller.rb

```diff
@@ -57,8 +57,8 @@ def decorated_user
     @_decorated_user ||= current_user.decorate
   end
 
-  def after_sign_in_path_for(user)
-    stored_location_for(user) || profile_path
+  def after_sign_in_path_for(_current_user = nil)
```

Comment:
this is a method in devise that we are overriding: https://github.com/plataformatec/devise/blob/88724e10adaf9ffd1d8dbfbaadda2b9d40de756a/lib/devise/controllers/helpers.rb#L213

super weird looking, I agree. I think a next step is to remove the inheritance from the devise controller so we are no longer needing to override all of these methods. It is confusing!